content/en-us/studio/plugins.md

```diff
@@ -42,13 +49,32 @@ newScriptButton.Click:Connect(onNewScriptButtonClicked)
 
 Plugins start from scripts. To create a plugin, create a `Class.Script` and save it as a plugin using the [Explorer](../studio/explorer.md). For example, to create the **EmptyScriptAdder Plugin**:
 
-1. Insert a new `Class.Script` inside **ServerStorage** and rename it to **EmptyScriptAdder**.
+1. Insert a new `Class.Script` inside `Class.ServerStorage` and rename it to **EmptyScriptAdder**.
 
    <img src="../assets/studio/plugins/Plugin-Empty-Script-Adder.png" width="320" />
 
 2. Copy and paste the **EmptyScriptAdder Plugin** code into the new script.
 3. Right-click the script in the Explorer and select **Save as Local Plugin**.
-4. In the popup window, click **Save** to insert the plugin script into your local **Plugins** folder of the Studio installation. The [Output](../studio/output.md) window indicates that the plugin successfully saved and the plugin runs for the first time after you save it.
+4. In the popup window, click **Save** to insert the plugin script into your local **Plugins** folder of the Studio installation.
+5. The plugin should appear in `Class.PluginDebugService` and start running.
+
+<Alert severity="warning">
+Make sure to delete the original script in `ServerStorage` and work from the
+plugin inside `PluginDebugService`, otherwise, you may end up applying changes
+to the wrong script.
```

Comment:
```suggestion
Make sure to delete the original script in `Class.ServerStorage` and work from the
plugin inside `Class.PluginDebugService`, otherwise, you may end up applying changes to the wrong script.
```